cmd/aries-agentd/startcmd/start.go

```diff
@@ -168,3 +193,12 @@ func startAgent(server server, host, inboundHost, dbPath string) error {
 
 	return nil
 }
+
+func startWebhookServer(webhookHost string) {
+	go func() {
+		webhookServerErr := webhook.StartWebhookServer(webhookHost, "/webhook")
```

Comment:
ACA-Py's websocket handler is registered on the `/ws` endpoint

Reply:
Will create a separate PR for websocket support. I'll be sure to register it on the /ws endpoint.